fips.md

```diff
@@ -0,0 +1,80 @@
+# FIPS mode
+
+## What is FIPS?
+
+According to [Wikipedia](https://en.wikipedia.org/wiki/Federal_Information_Processing_Standards) 
+> Federal Information Processing Standards (FIPS) are publicly announced standards developed by the National Institute of Standards and Technology for use in computer systems by non-military American government agencies and government contractors.
+> FIPS standards are issued to establish requirements for various purposes such as ensuring computer security and interoperability and are intended for cases in which suitable industry standards do not already exist.
+
+In short it is a thoroughly tested and verified set of standards which could be used to implement high level of security. In terms of software we are usually speaking specifically about FIPS 140-2. 
+
+ReadonlyREST uses OpenSource [BouncyCastle](https://www.bouncycastle.org) library to provide FIPS 140-2 compliant algorithms.
+
+## Does ReadonlyREST is fully FIPS compliant?
+
+Unfortunately not yet. Currently only SSL transport part uses FIPS compliant algorithms.
```

Comment:
we can confiture transport SSL (internode one, 9300) and http SSL (9200) in ES. Maybe "SSL transport" can be misleading for a user?

Reply:
I improved it a little